javascript-data-model/main.js

```diff
@@ -0,0 +1,28 @@
+/* exported orderHistory */
+
+var orderHistory = [
+  {
+    itemNames: 'JavaScript for Impatient Programmers',
+    type: 'book',
+    returnDeadline: 'September 7, 2020',
+    orderNumber: '114-394'
+  },
+  {
+    itemNames: 'The Timless Way of Building',
+    type: 'book',
+    returnDeadline: 'August 19, 2020',
+    orderNumber: '113-998'
+  },
+  {
+    itemNames: 'Gamecube Controller Adapter',
+    type: 'game acessories',
+    returnDeadline: 'August 5, 2020',
+    orderNumber: '114-287'
+  },
+  {
+    itemNames: ['Gamecube Controler for Smash', 'The art of SQL'],
```

Comment:
And here, the `itemNames` property holds an array of strings. If the data type being held in a property does not match between multiple entries in a data model, working with each piece of data in a loop becomes nearly impossible. Even if the data is different, the format of your data model should always be the same.